src/shared/ui/upload/molecules/FileCard.tsx

```
@@ -34,16 +47,71 @@ export const FileCard = ({
+          className
+        )}
+        role={onClick ? 'button' : undefined}
+        tabIndex={onClick ? 0 : undefined}
+        onClick={onClick}
+        onKeyDown={onClick ? (event) => {
+          if (event.key === 'Enter' || event.key === ' ') {
+            event.preventDefault();
+            onClick();
+          }
+        } : undefined}
+      >
```

ESLint

[error] Avoid non-native interactive elements. If using native HTML is not possible, add an appropriate role and support for tabbing, mouse, keyboard, and touch inputs to an interactive content element.

(jsx-a11y/no-static-element-interactions)